tee-worker/docker/lit-substrate-di-test.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  lit-substrate-di-test:
+    image: litentry-cli:dev
+    container_name: litentry-substrate-di-test
+    volumes:
+      - ../ts-tests:/ts-tests
+      - ../cli:/usr/local/worker-cli
+    build:
+      context: ..
+      dockerfile: build.Dockerfile
+      target: deployed-client
+    depends_on:
+      litentry-node:
+        condition: service_healthy
+      litentry-worker-1:
+        condition: service_healthy
+      litentry-worker-2:
+        condition: service_healthy
+    networks:
+      - litentry-test-network
+    entrypoint:
+      "bash -c '/usr/local/worker-cli/lit_ts_test.sh substrate-di-examples 2>&1' "
```

Comment:
This won't work if `substrate-di-examples:staging` isn't defined (and configured with the right `.env` file, etc) in `tee-worker/ts-tests/integration-tests/package.json`.

Not sure if it's worth to add this tbh; the `identity` test suite has already been ported and the `vc` one is [underway](https://github.com/litentry/litentry-parachain/pull/1956); these examples [should become redundant](https://github.com/litentry/litentry-parachain/issues/1987#issuecomment-1665735488) then.